postfix/values.yaml

```diff
@@ -465,7 +467,7 @@ postfix:
       # In the left-hand side, specify an @domain.tld wild-card, or specify
       # a user@domain.tld address.
       #
-      #relay_recipient_maps = hash:/etc/postfix/relay_recipients
+      #relay_recipient_maps = lmdb:/etc/postfix/relay_recipients
```

Comment:
`cyrus-sasl-plain` is no longer in alpine, so it has been changed.
https://github.com/juanluisbaptiste/docker-postfix/issues/45